### PR TITLE
More brands (most of them from Spain)

### PR DESCRIPTION
**Pull request by ArenaL5**
Commits should be self-explanatory. There's no theme to this PR as a whole, it's three separate commits affecting several files (`office/insurance` and `shop/*`).

### CODE REVIEW

File: brands/shop/newsagent.json [hunk 1]
```diff
@@ -11,6 +11,8 @@
   "shop/newsagent|Hudson News": {
     "tags": {
       "brand": "Hudson News",
+      "brand:wikidata": "Q5928787",
+      "brand:wikipedia": "en:Hudson News",
```
Comment (tas50) [inline brands/shop/newsagent.json]: I’m not sure if these are valid anymore. Hudson News has rebranded every location I’ve been to across the US lately to just Hudson.

Comment (ArenaL5) [reply brands/shop/newsagent.json]: I don't know that brand by experience, I just found it on Wikidata.

Would this be okay, instead?

```
  "shop/newsagent|Hudson": {
    "matchNames": ["Hudson News"],
    "tags": {
      "brand": "Hudson",
      "brand:wikidata": "Q5928787",
      "brand:wikipedia": "en:Hudson News",
      "name": "Hudson",
      "shop": "newsagent"
    }
  },
```

(Maybe we should change the WIkipedia article too, but that's the less important data. It's currently a redirection anyways.)

**Comment by bhousel** [comment]
Thanks @ArenaL5 these look great 👍 
(I agree we should switch Hudson News -> Hudson.. they rebranded as of only a few months ago)

BTW thanks for all your contributions recently! To speed things up, I've invited you to be an admin on this project.  Feel free to triage issues, merge PRs, and commit directly to the master branch anything that you feel good about. 🙇

File: brands/shop/newsagent.json (hunk 1 above)
Comment (bhousel) [reply brands/shop/newsagent.json]: in 1b5ea1777 I changed it to this:
```js
  "shop/newsagent|Hudson": {
    "matchNames": ["hudson news"],
    "tags": {
      "brand": "Hudson",
      "brand:wikidata": "Q5928682",
      "brand:wikipedia": "en:Hudson Group",
      "name": "Hudson",
      "shop": "newsagent"
    }
  },
```

**Comment by ArenaL5** [comment]
I don't think I have helped _that_ often, but if you're okay with my job so far, I'd be glad to join. Thank you. :slightly_smiling_face:

**Pull request merged**